omnibus/package-scripts/agentless-scanner-deb/postinst

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+#
+# Perform necessary datadog-agentless-scanner setup steps after package is installed.
+#
+# .deb: STEP 5 of 5
+
+INSTALL_DIR=/opt/datadog/agentless-scanner
+LOG_DIR=/var/log/datadog
+CONFIG_DIR=/etc/datadog-agent
+SERVICE_NAME=datadog-agentless-scanner
+
+# If we are inside the Docker container, do nothing
+if [ -n "$DOCKER_DD_AGENT" ]; then
+    echo "Installation from docker-dd-agent, nothing to do in postinst"
+    exit 0
+fi
+
+set -e
+case "$1" in
```

Comment:
Since the agentless scanner is supposed to depend on the agent, is this required?

I'd expect the datadog-agent package to have done this already

Reply:
I've removed the creation of `dd-agent` user, `dd-agent` group and the chmod.